crates/utils/sov-test-utils/src/runtime/macros.rs

```diff
@@ -165,6 +165,15 @@ macro_rules! generate_runtime_without_capabilities {
                 }
             }
 
+            fn resolve_addrss<ST: sov_modules_api::StateAccessor>(
+                &mut self,
+                default_address: &S::Address,
+                credential_id: &sov_modules_api::CredentialId,
+                state: &mut ST,
+            ) -> Result<S::Address, <ST as sov_modules_api::StateWriter<sov_modules_api::User>>::Error>{
```

Comment:
Do we actually need this StateWriter::error? Do we do anything with it? Maybe anyhow will be enough.

Reply:
I think in this case caller should decide. It is always possible to convert it on the caller side. In our case we just unwrap because StateCheckPoint can't run out of gas. But this method can work with anys state accessor

Reply:
Ok, I see.

Another thing, we don't we use `::` here like in other methods

```suggestion
            ) -> Result<S::Address, <ST as ::sov_modules_api::StateWriter<::sov_modules_api::User>>::Error> {
```

Reply:
fixed